cadvisor.go

```diff
@@ -35,6 +36,7 @@ import (
 	"github.com/golang/glog"
 )
 
+var argPath = flag.String("listen_path", "", "Path to listen on (UNIX socket), defaults to empty (use TCP instead)")
```

Comment:
It's not clear from the description that this overrides listening on TCP. Maybe we should deprecate `listen_ip` and `port`, and switch to a similar approach to docker, a list of paths / URIs to listen to (e.g. `--listen=tcp://localhost:8080,unix:///var/run/cadvisor.sock`)